variables-cluster.tf

```diff
@@ -8,7 +8,7 @@ variable "create_cluster" {
 }
 
 variable "cluster_name" {
-  default     = "oke"
+  default     = null
```

Comment:
Small fix to properly utilize `cluster_name` default with `state_id` suffix added previously.